clang/lib/Basic/NoSanitizeList.cpp

```diff
@@ -44,7 +44,8 @@ bool NoSanitizeList::containsFunction(SanitizerMask Mask,
 
 bool NoSanitizeList::containsFile(SanitizerMask Mask, StringRef FileName,
                                   StringRef Category) const {
-  return SSCL->inSection(Mask, "src", FileName, Category);
+  bool allowList = SSCL->inSection(Mask, "src", FileName, "sanitize");
```

Comment:
I guess we need to move "sanitize" handling from ASTContext::isTypeIgnoredBySanitizer in here.

Reply:
Probably don't do this yet.
Let's resolve for file, and then expand to the rest.